#
# If /etc/localtime is a symlink, `test -r` fails and we get
# http://bugs.sourcemage.org/show_bug.cgi?id=14548
#
# Signed-off-by: Eric Sandall <sandalle@sourcemage.org>
#
diff -Naur glibc-2.7.orig/timezone/Makefile glibc-2.7/timezone/Makefile
--- glibc-2.7.orig/timezone/Makefile	2008-10-12 23:17:37.971435756 -0700
+++ glibc-2.7/timezone/Makefile	2008-10-12 23:27:14.785187504 -0700
@@ -144,7 +144,7 @@
 $(installed-localtime-file): $(inst_zonedir)/$(localtime) $(objpfx)zic \
 			     $(+force)
 	$(make-target-directory)
-	if test -r $@; then \
+	if test -r $@ || test -h $@; then \
 	  echo Site timezone NOT reset to Factory.; \
 	else \
 	  rm -f $@T; \
